Algorithms/Hard/4_MedianofTwoSortedArrays/Solution.java

```diff
@@ -0,0 +1,40 @@
+class Solution {
+    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
+        double median = 0.0;
+        int n1 = nums1.length;
+        int n2 = nums2.length;
```

Comment:
Please change the code to this:
```java
int n = nums2.length;
```
in order to keep consistent with the problem description. Also, change in other places where you've used `n2`.